src/gleam/utf.gleam

```diff
@@ -0,0 +1,62 @@
+/// Splits a non-empty `String` into its head and tail. This lets you
```

Comment:
Sorry but we do not want to create a new module here, especially one that is indistinguishable from the string module. Users now have 2 modules to look in when dealing with strings